tests/unit/Authentication.spec.js

```diff
@@ -87,10 +87,10 @@ describe('Authentication', () => {
     expect(wrapper.vm.otpFormVisible).toBe(true);
     wrapper.setData({ otp: 'foobar' });
     await flushPromises();
-    expect(wrapper.vm.otp).toBe(''); // Test masking
+    expect(wrapper.vm.$refs.login.inputs[0].validate()).toBe(false);
```

Comment:
Note: Behavior changed in Vuetify, invalid input won't be removed automatically.